tasks/Resources/Configurations/groups/rhods-groups-config.resource

```diff
@@ -0,0 +1,39 @@
+*** Settings ***
+Documentation    Keywords for task RHODS groups configuration
+
+
+*** Variables ***
+${DEFAULT_ADMIN_GROUPS} =       dedicated-admins
+${DEFAULT_ALLOWED_GROUPS} =     system:authenticated
+${RHODS_GROUPS_CONFIG} =        tasks/Resources/Configurations/groups/rhods_groups_config.yaml
+
+
+*** Keywords ***
+Modify RHODS Groups
```

Comment:
- This keyword should be in tests/Resources/ODS.robot
- I think it's better to rename it to `Modify ODS Groups`
- [Documentation] tag should also explain better what this keyword is doing (e.g. customize ODS admin and allowed groups)
- You can also add a Restore Default ODS Groups, restoring the default configuration

Reply:
Point one is discussed in your last comment.
Points two and three are done. 
Point four, a restore keyword is not necessary because this is a task that will be used to change the groups dynamically, and there is no need to restore the default values after, as how it would happen in a test